src/components/Cards/CardLarge/CardLarge.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import './CardLarge.scss';
+
+const CardLarge = ({ image, imageAlt, header, paragraph, button }) => {
+  return (
+    <div className='card-large'>
+      <img className='card-large__image' src={image} alt={imageAlt}/>
+      <div className='card-large__text'>
+        <p className='card-large__header'>{header}</p>
+        <p className='card-large__paragraph'>{paragraph}</p>
+      </div>
+      <button className='card-large__button'>{button}</button>
```

Comment:
My Button PR has been merged. You can incorporate the Button component if you'd like.

Reply:
Button has been added to all cards.